SteakBot.Core/Modules/MemeModule.cs

```diff
@@ -20,20 +20,50 @@ public MemeModule(MemeService memeService)
         public async Task AddMeme(string name, string value, string description)
         {
             var replyMessage = string.Empty;
-            EmbedBuilder embed = null;
+            EmbedBuilder embed = new EmbedBuilder();
 
-            var type = value.ToLower().Contains("http") ? MemeResultType.Image : MemeResultType.Text;
+            var type = _memeService.GetMemeType(value);
 
             var result = _memeService.AddCommand(new MemeCommand(type, name, value, description));
 
-            if (result)
+            await SendReply(result, type, value, "Add failed! ;(");
+        }
+
+        [Command("editMeme")]
+        public async Task EditMeme(string oldName, string newName, string newValue, string newDescription)
+        {
+            var replyMessage = string.Empty;
+            EmbedBuilder embed = new EmbedBuilder();
```

Comment:
Both local fields are now unused and obsolete.

Reply:
Corrected.